app/views/curation_concerns/file_sets/_single_use_link_rows.html.erb

```diff
@@ -0,0 +1,18 @@
+<% single_use_links.each_with_index do |link, index| %>
+  <% type = link.path =~ /downloads/ ? 'Download' : 'Show' %>
+  <% url_helper = link.path =~ /downloads/ ? 'download_single_use_link_url' : 'show_single_use_link_url' %>
+  <tr>
+    <td><%= type %></td>
+    <td><%= link.downloadKey %></td>
```

Comment:
It exposes everything, but the distinction isn't really explained. If we only want to show one kind of link, I would argue that it should be the "show" link, although we don't have to name it as such. In the process of visiting a show link, a download link is created automatically. However, if the user visited the show link, but never actually downloaded the content, the auto-created download link would still be hanging around. The user might want to be able to delete it, if they felt the need to control access, but no one would know the key of that download link anyway, so it's probably a moot issue.